### DISCUSSION TITLE
Galaxy: encrypt settings.json and enable adding new ones

### DISCUSSION THREAD

**Discussion by satyavh** [meteor/meteor#11520] [Feature Requests]
It just hurts to see all the secrets out there in the dashboard. For example, Render provides a way (similar to Gitlab) to manually add secrets in their dashboard and restart the server upon changes, and have those secrets displayed obfuscated.

This would also help that you don't need to redeploy to update a single settings (facing the annoyingly slow build process again just for changing a setting)

**Comment by StorytellerCZ** [comment]
If I remember correctly it is possible to change the app settings in Galaxy.

**Comment by filipenevola** [comment]
Hi @satyavh, you are not required to expose your settings on Galaxy, if you want we have a flag that you can set in your account and then your settings are going to be hidden from the dashboard.

This is explained [here](https://galaxy-guide.meteor.com/environment-variables.html#settings-example):
> You can contact support to obscure your settings.json file. Once this is done, all app settings will be hidden across your account. A user with access to any of your app’s dashboards will not be able to view the information in your settings.json file from within the associated dashboard after this feature is enabled.

Also as @StorytellerCZ said you can change your settings directly in the current version, you don't need to run the full deploy to change settings on Galaxy.

**Reply by satyavh** [reply]
> you can change your settings directly in the current version

I don't see that option in the Galaxy UI?

**Reply by filipenevola** [reply]
Inside Galaxy: Go to your App, go to the Versions tab, expand your current Version and you are going to see an edit button below your current settings. 😉
